### DISCUSSION TITLE
Table with many columns

### DISCUSSION THREAD

**Discussion by dkamm** [TanStack/table#2814] [Q&A] [answered]
I have a table with date columns that looks like this

| | 10/9 | 10/8 | 10/7 | ... |
| --- | --- | --- | --- | --- |
| row 1| | | | |
| row 2| | | | |

My `data` looks like so
```
[{
    name: "row 1",
    data: [1, 2, 3, 4, ...]
  },
  {
    name: "row 2",
    data: [2, 3, 4, 5, ...]
  }
]
```
and I was able to get a basic version working with a functional `accessor`.  

Since the number of columns can be large (multiple months/year), I would like to have horizontal scrolling too.

Is this outside the intended use case of `react-table`?  If my rows and columns were reversed, this would be easily handled.  But I want to keep it this way for presentational purposes.

**Comment by dkamm** [comment, marked as answer]
I realized my use case is actually a data sheet

**Reply by ankursehdev** [reply]
are you still using this library for this kind of data? or did you find any other lib to work in this case? I am dealing with same kind of data sets.

**Reply by dkamm** [reply]
I did not end up using this library for that use case. I ended up using regular `<table>` elements and it worked fine for me.
